Azure.OpenAI/Client/Extensions/Localization.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) David Pine. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.OpenAI.Client.Extensions;
+
+internal sealed class Localization
```

Comment:
```suggestion
internal static class Localization
```

Extensions have to be in static classes.

Reply:
Yeah 
I fixed it in forked code.